tls.h

```diff
@@ -290,6 +294,9 @@ struct tls_context {
 	struct list_head list;
 	refcount_t refcount;
 	struct rcu_head rcu;
+
+	lua_State *L;
+	struct tls_lua_info lua_info;
```

Comment:
The Lua state has everything we need.
We only need to keep one more extra information, which is the entry point to call on the `recv()` system call.